src/display.rs

```diff
@@ -162,6 +165,30 @@ impl Display for RawTransaction {
     }
 }
 
+impl Display for NodeInfo {
+    fn to_json(&self) -> Json {
+        let mut info_pair = Map::new();
+        info_pair.insert(
+            String::from("address"),
+            Json::from(format!("0x{}", hex::encode(&self.address))),
+        );
+        let net_info = self.net_info.as_ref().unwrap();
+        info_pair.insert(String::from("origin"), Json::from(net_info.origin));
+        let multi_address: Multiaddr = net_info.multi_address[..].parse().unwrap();
+        for i in multi_address.iter() {
+            let info_string = format!("{}", i);
```

Comment:
There are some methods for parsing those protocols in `tentacle_multiaddr`. 

[example](https://github.com/cita-cloud/network_tls/blob/763a7341d821d5d369c192414ada4eb4881517bf/src/server.rs#L499-L524)

Reply:
adopted, thanks